modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/api.ts

```diff
@@ -147,3 +153,9 @@ export async function getAllPicklists(
 
 	return items;
 }
+
+export function isSortable(field: IFDSField | IField): boolean {
+	return !(
+		field.type === EFieldType.OBJECT || field.type === EFieldType.ARRAY
```

Comment:
Handling array type of fields is technically not in the scope of the task. Based on @dsanz [research](https://liferay.atlassian.net/wiki/spaces/ENGFRONTENDINFRA/pages/2739274471/LPD-17838+-+Reseach+sorting+capabilities+for+leaf+elements), there are some array fields where sorting works, e.g. `cmultiselectpicklist`. Do we want to disable them all?